doc/other_distributions.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
Should this rather be included in a toc somewhere?

Reply:
Could be, maybe in [basic install instructions](https://scikit-learn.org/stable/install.html).
But then you have a page with only one entry in the toc. At this stage a simple link is an acceptable solution.
IMO, `install.rst` and `other_distributions` both contain information about other distributions and they are not really up to date on that point... so both need to be revamped... May I fix this in the next PR? :)

Reply:
Actually, since we now have a "third party distributions" section in install.rst, I don't really see why this needs to be separate.

Reply:
As `other_distributions` needs to be updated a bit, putting the pages together will probably give you a file a bit "long", but if you are not afraid of that I tend to agree with you.
Will open a PR in that direction just after merging this one... :) ...

Reply:
+1 to move it as a separate section at the end of https://scikit-learn.org/dev/install.html

Reply:
Ok, got it. See #15011 .